README.md

```diff
@@ -1,7 +1,10 @@
-# Verifiable Credentials JS Library _(vc-js)_
-
-[![Build Status](https://travis-ci.org/digitalbazaar/vc-js.png?branch=master)](https://travis-ci.org/digitalbazaar/vc-js)
-[![NPM Version](https://img.shields.io/npm/v/vc-js.svg?style=flat-square)](https://npm.im/vc-js)
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>
+  Verifiable Credentials JS Library (react-native-vc-js)
```

Comment:
I think we can say something about React-Native in here. We should make the README title in line with what the other libraries say. For instace, `react-native-credentials-context` says: A React Native supported W3C Credentials Context Library

Reply:
done